src/doc/03 - security.rst

```diff
@@ -16,7 +16,206 @@ links between the records in different subsystems`
+    Create Person                       ``pr.person.write``
+    Read Person                         ``pr.person.read``
+    Update Person                       ``pr.person.write``
+    Delete Person                       ``pr.person.write``
+    Merge Persons                       ``pr.person.write``
+    Create Identity                     ``pr.identity.write``
+    Read Identity                       ``pr.identity.read``
+    Update Identity                     ``pr.identity.write``
+    Partial Update Identity             ``pr.identity.write``
+    Delete Identity                     ``pr.identity.write``
+    Set Identity Status                 ``pr.identity.write``
+    Define Reference                    ``pr.reference.write``
+    Read Reference                      ``pr.reference.read``
+    Read Galleries                      ``pr.gallery.read``
+    Read Gallery Content                ``pr.gallery.read``
+    ----------------------------------- -----------------------------------------------
+    **Biometrics**
+    -----------------------------------------------------------------------------------
+    Create Encounter                    ``abis.encounter.write``
+    Read Encounter                      ``abis.encounter.read``
+    Update Encounter                    ``abis.encounter.write``
+    Delete Encounter                    ``abis.encounter.write``
+    Merge Encounter                     ``abis.encounter.write``
+    Set Encounter Status                ``abis.encounter.write``
+    Read Template                       ``abis.encounter.read``
+    Read Galleries                      ``abis.gallery.read``
+    Read Gallery content                ``abis.gallery.read``
+    Identify                            ``abis.identify``
+    Verify                              ``abis.verify``
+    ----------------------------------- -----------------------------------------------
+    **Credential Services**
+    -----------------------------------------------------------------------------------
+    Create Credential Request           ``cms.request.write``
+    Read Credential Request             ``cms.request.read``
+    Update Credential Request           ``cms.request.write``
+    Delete Credential Request           ``cms.request.write``
+    Read Credential                     ``cms.credential.read``
+    Suspend Credential                  ``cms.credential.write``
+    Unsuspend Credential                ``cms.credential.write``
+    Cancel Credential                   ``cms.credential.write``
+    ----------------------------------- -----------------------------------------------
+    **ID Usage**
+    -----------------------------------------------------------------------------------
+    Verify ID                           ``id.verify``
+    Identify                            ``id.identify``
+    Read Attributes                     ``id.read``
+    Read Attributes set                 ``id.set.read``
```

Comment:
The initial idea, I think, was to have a "read" giving access to any attribute and a "read set" with a narrower scope: hence the two scopes.
Since this part if being reviewed/redesigned under the leadership of Thalès, I suggest we keep it out this PR for the moment.